code/game/machinery/_machinery.dm

```diff
@@ -120,16 +120,17 @@ Class Procs:
 	var/speed_process = FALSE			//If false, SSmachines. If true, SSfastprocess.
 
 /obj/machinery/Initialize(mapload, newdir)
+	.=..()
```

Comment:
are you sure this is a good idea?
sometimes, stuff depends on initialization proc call order being in a certain way.

Reply:
Made the change back, just in case